### PR TITLE
Implementation for Service accounts

### PR DESCRIPTION
**Pull request by prashanthca**
This code change will allow the usage of service accounts with GDIndex. This will mainly benefit the end-users

### CODE REVIEW

**Comment by maple3142** [comment]
Looks good, but I don't really what this is for. Also, add a brief documentation to `README.md` might be necessary, or users will not know this feature.

As for code addition, please fix some coding style using prettier with [this](https://github.com/maple3142/dotfiles/blob/master/.prettierrc) config. Also, prefer `const` over `let`, `var` if not necessary.

**Comment by CatTheHacker** [comment]
Service Accounts can be used instead of standard Google/GSuite accounts with `rclone` setup.
The same way `rclone` can use Service Accounts - https://rclone.org/drive/#service-account-support.

**Comment by prashanthca** [comment]
Thanks for looking into this @maple3142! And yes, as @CatTheHacker said, it's basically a way to use service accounts to access GDrive. This eliminates the standard upload/download quota that usually applies to normal accounts

**Comment by CatTheHacker** [comment]
> This eliminates the standard upload/download quota that usually applies to normal accounts

Service accounts are still subject to upload/download quota just like the normal user accounts, it's just easier and faster to manage/use as they don't require manual authentication via web browser.

**Comment by prashanthca** [comment]
@CatTheHacker Yes, it seems that I was mistaken about the quota. Nevertheless, as you mentioned, it's a easier way to setup GDIndex, without having to get the refresh token via rclone or the web method. Maybe I can take a look into modifying the props to contain multiple service account credentials, so the client can cycle through them randomly, distributing the usage across the accounts.

**Comment by prashanthca** [comment]
@maple3142, I've made the changes you suggested and I've added a section in README for this feature. I've made changes in the Chinese README (both traditional and simplified versions) files as well.

File: worker/package.json [hunk 1]
```diff
@@ -5,10 +5,14 @@
...
 		"mime": "^2.4.4",
-		"path-to-regexp": "^3.1.0"
+		"path-to-regexp": "^3.1.0",
+		"buffer": "*",
+		"@sagi.io/workers-jwt": "*"
 	},
 	"devDependencies": {
-		"bili": "^4.8.1"
+		"bili": "^4.8.1",
+		"rollup-plugin-node-builtins":"*",
```
Comment (maple3142) [inline worker/package.json:14]: Please don't use "*" as version number.
Just use what appears after running `npm install -S $pkg` or `yarn install $pkg`

**Review by maple3142 — CHANGES_REQUESTED** [review]

File: worker/package.json (hunk 1 above)
Comment (maple3142) [reply worker/package.json:14]: Others packages using "*" should be fixed as well. As long as this is fixed, I can merge that PR now.

Comment (prashanthca) [reply worker/package.json]: I've pushed the fix

**Pull request merged**